engine-rest/engine-rest-openapi/src/main/templates/paths/history/batch/count/get.ftl

```diff
@@ -0,0 +1,57 @@
+<#-- Generated From File: camunda-docs-manual/public/reference/rest/history/batch/get-query-count/index.html -->
+<#macro endpoint_macro docsUrl="">
+{
+  <@lib.endpointInfo
+      id = "getHistoricBatchesCount"
+      tag = "Historic Batch"
+      summary = "Get Historic Batch Count"
+      desc = "Requests the number of historic batches that fulfill the query criteria.
+              Takes the same filtering parameters as the
+              [Get Historic Batches](${docsUrl}/reference/rest/history/batch/get-query/)
+              method."
+  />
+
+  "parameters" : [
+
+    <#assign last = true >
+    <#assign requestMethod="GET"/>
```

Comment:
I think we don't need the request method at the moment since we don't use this in the query-params file.